package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/kriskuiper/project-tech#readme",
   "devDependencies": {
-    "eslint": "^5.15.1",
+    "eslint": "^5.16.0",
```

Comment:
Why did you update ESLINT?

Reply:
I've got no clue. I probably accidentally used npm install, which is why it was probably updated.